Mongo.Migration/Migrations/Locators/MigrationLocator.cs

```diff
@@ -77,7 +77,8 @@ public DocumentVersion GetLatestVersion(Type type)
         {         
             var migrations = GetMigrations(type);
 
-            return migrations.Max(m => m.Version);
+            return migrations?.Max(m => m.Version)
+                ?? new DocumentVersion(0,0,0);
```

Comment:
Use DocumentVersion default for this. Its a struct. 

` return migrations?.Max(m => m.Version) ?? DocumentVersion.Default();`

Reply:
I needet to add this line of code to fix another issue. You can remove it from the PR.